src/buf/uninit_slice.rs

```diff
@@ -22,42 +22,42 @@ use core::ops::{
 pub struct UninitSlice([MaybeUninit<u8>]);
 
 impl UninitSlice {
-    /// Creates a `&mut UninitSlice` wrapping slice of uninitialised memory.
+    /// Creates a `&mut UninitSlice` wrapping slice of initialised memory.
     ///
```

Comment:
This sentence doesn't seem grammatical to me.

Reply:
You could say "wrapping a slice".

Reply:
Done.